master/internal/trial_workload_sequencer.go

```diff
@@ -9,8 +9,10 @@ import (
 )
 
 type stepInfo struct {
-	hasValidation bool
-	hasCheckpoint bool
+	hasValidation    bool
+	hasCheckpoint    bool
+	batchesPerStep   int
```

Comment:
In keeping with the request to rename the `model.Step.BatchesPerStep`, I'd also like to see this renamed.